jenkins-jobs/pipelines/release-pipeline.groovy

```diff
@@ -525,12 +525,16 @@ node('Slave') {
                 }
             }
             dir("$IMAGES_DIR") {
-                modifyFile('build-all.sh') {
+                modifyFile('build-all-multiplatform.sh') {
                     it.replaceFirst('DEBEZIUM_VERSION=\"\\S+\"', "DEBEZIUM_VERSION=\"$IMAGE_TAG\"")
                 }
             }
             dir(IMAGES_DIR) {
-                sh "env SKIP_UI=true ./build-all.sh"
+                sh """
+                    docker run --privileged --rm tonistiigi/binfmt --install all
+                    ./setup-local-builder.sh
```

Comment:
@jcechace Please make sure that release process does not depend on ARM images being built